docs/usage/templates.md

```diff
@@ -30,3 +30,11 @@ If you want to print pretty JSON with Handlebars you can use the built-in functi
 `{{{stringToPrettyJSON myvar}}}`
 
 In the example above `myvar` is a variable/field, that contains valid JSON.
+
+### encodeURIComponent
+
+If you want to ..., use the built-in function `encodeURIComponent` like this:
```

Comment:
I need help to explain what `encodeURIComponent` does, and need an example on how to use it.

Reply:
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent